Automattic/wp-calypso
# Update social previews to use npm package components (again)

This is basically another attempt to ship https://github.com/Automattic/wp-calypso/pull/44351. Please see the description of that PR for full explanation.

Fixes https://github.com/Automattic/wp-calypso/issues/44605.


#### Testing instructions

- Run through testing instructions in https://github.com/Automattic/wp-calypso/pull/44351.
- Repeat the same tests steps on the [Calypso live link](https://github.com/Automattic/wp-calypso/pull/44604#issuecomment-667910080) - particularly note that the CSS styles are **_correctly applied_** to the components within the social preview as this was [the original error that necessitated the revert of #44351](https://github.com/Automattic/wp-calypso/pull/44381).
